packages/sdk/src/wallet/providers/privy.ts

```diff
@@ -55,8 +56,8 @@ export class PrivyWalletProvider implements WalletProvider {
       // TODO: Implement proper user-to-wallet lookup
```

Comment:
I think this TODO might still be true and need work. If I initialize a wallet with ID `123` and later pass `123` to privy's getWallet function, I _think_ it'll error because they expect a walletId and not our arbitrary userId.